Add a Bounty Proposal for Implementing Claimable Balances in the Account Viewer (#115)
tyvdh/stellar-quest-bounties: bounties/level-1/claimable_balances_account_viewer.md

# Pending Claimable Balances in the Stellar Account Viewer
## Award
| Hunter | Reviewer | Type
| :- | :- | :-

> | 1000 XLM | 500 XLM | Competitive | 

Reviewer: Let's do 1500 XLM and 800 XLM, this one will be a little more difficult.

## Description
The [Stellar Account Viewer](https://accountviewer.stellar.org/) is a great tool that allows you to see how many XLM you own, the transactions you received and the option to send payments. Best of all, it's Open Source and anyone can send changes. You can find it on [GitHub](https://github.com/stellar/account-viewer-v2)

### What is this task?
Your goal is to create a new section of the account viewer designated to Claimable Balances. 

This bounty can be separated into 2 Main objectives.

1) List Claimed Balances in the Payment History or a separate new tab

> 2) New section that will list all pending claimable balances and allow for their claimant

Reviewer 2: ```suggestion
2) New section that will list all pending claimable balances and allow for claiming them.
```

# Pending Claimable Balances in the Stellar Account Viewer
## Award
| Hunter | Reviewer | Type
| :- | :- | :-

> | 1500 XLM | 800 XLM | Competitive | 

Reviewer: ```suggestion
| 1500 XLM | 800 XLM | Capped (1) | 
```

# Pending Claimable Balances in the Stellar Account Viewer
## Award
| Hunter | Reviewer | Type
| :- | :- | :-
| 1000 XLM | 500 XLM | Competitive | 

## Description

> The stellar account viewer is a great tool that allows you to see how many XLM you own, the transactions you received and the option to send payments.

Reviewer 2: Since the account viewer is so central to this bounty, we might want to have links to the account viewer website and repository up here, as well.

### What is this task?
Your goal is to create a new section of the account viewer designated to Claimable Balances. 

This bounty can be separated into 2 Main objectives.

1) List Claimed Balances in the Payment History or a separate new tab
2) New section that will list all pending claimable balances and allow for their claimant

### What are the requirements for the bounty hunter?
For this bounty to be completed a Pull Request on the following repo is required: [GitHub - stellar/account-viewer-v2: A simple tool to view an account on the Stellar network and make transactions from it.](https://github.com/stellar/account-viewer-v2) with your changes.

> The Account Viewer is built with React and Typescript. Along with that keep in mind of the existing design guidelines and make sure it's understandable to the user.

Reviewer 2: ```suggestion
The Account Viewer is built with React and TypeScript. Along with that keep in mind the existing design guidelines and make sure it's understandable to the user.
```

### What are the deliverables?

#### Bounty Description
- The hunter should provide a link to the Pull Request containing the solution.

#### Submission Procedure

> - The hunter submits a pull request to account-viewer-2

Reviewer 2: ```suggestion
- The hunter submits a pull request to the `account-viewer-2` repository.
```

> - When the Pull Request is merged only then is the bounty completed.

Reviewer 2: ```suggestion
- The bounty will only be considered complete, after the Pull Request has been merged into the main repository.
```

I think this makes it a bit easier to read?

- Any issues they need to be aware of?

> - Any third parties they need to communicate with?

Reviewer 2: I think these lines got left in from the bounty template.

> - Any third parties they need to communicate with?

Reviewer 2: Does somebody need to communicate with the account-viewer-2 maintainer? Are we sure they're open to this feature? (Just asking for clarity's sake Hehehe)
